dev/scripts/start_depselector.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+#
+# Start depselector in the forground in order to see any error
+# messages it is producing.  This is useful in some debugging
+# situations.
+#
+# The erlang runtime would open these FDs for us. If we don't open
+# them they get reserved by the ruby VM and then application fails to
+# start.
+exec 3<&0
+exec 4>erlout
```

Comment:
`n>>word` does indeed work in Bash.  Happy to use that, however I'm not sure if appending random bytes is really better than overwriting. We could use a temporary file I suppose.